invenio_communities/communities/records/mappings/v7/communities/communities-v1.0.0.json

```diff
@@ -51,19 +51,16 @@
             "type": "text"
           },
           "organizations" : {
-            "properties" : {
-              "identifiers" : {
-                "properties": {
-                  "identifier" : {
-                    "type" : "keyword"
-                  },
-                  "scheme" : {
-                    "type" : "keyword"
-                  }
-                }
+            "type": "object",
```

Comment:
missing changes for funding?

Reply:
See also https://github.com/inveniosoftware/invenio-rdm-records/pull/936/files#diff-77181e11a2fb6a1736c1ce171f3192f5fc365850af9840218a8c0a84eff304d4